### PR TITLE
Portfolio: desktop version

### PR DESCRIPTION
**Pull request by Innocent259**
In this project I:

1.  Created a desktop version of My portfolio using media queries. 
2.  Created some styles just to meet the Figma design and responsive web page to both mobile and desktop versions.

### CODE REVIEW

**Review by jerryowusu — COMMENTED** [review]
Hi @Innocent259 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

Your project is missing the Future section which is a key part of the readme file. Kindly make sure that all sections of the readme file is present. 

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by Innocent259** [comment]
Hi @jerryowusu,
Thank you! I did it but my project is not being reviewed spent more than 2 hours

File: index.html [hunk 1]
```diff
@@ -2,83 +2,103 @@
...
-    <body class="container">
-        <section class="first-section">
-            <header>
-            <div class="nav">
-                <a href="#" class="nav-title">Innocent N.</a>
-                <nav>
-                    <img src="./images/menu.png" alt="menu-icon" class ="nav-menu" />
-                </nav>      
-            </div>
+    <body>
```
Comment (hiromataba) [inline index.html:13]: - Please get the feedback from the original design in the [video](https://youtu.be/avnyFbbjxXI) 😉  I think you find this very helpful

**Review by hiromataba — CHANGES_REQUESTED** [review]
Hi @Innocent259    :wave: :wave: 

almost-there

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- Your PR has a proper title and description  :+1: 
- Your linters are passing :heavy_check_mark:  
- Profesional README file :rocket: 

## Required Changes ♻️

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css [hunk 2]
```diff
@@ -1,12 +1,17 @@
...
+* {
   box-sizing: border-box;
+}
+
+body {
+  height: 100%;
   margin: 0;
-  min-width: 375px;
+  width: fit-content;
 }
```
Comment (Owusu-Desmond) [inline style.css:9]: - [ ] As requested by the previous code reviewer, your project has a horizontal scroll on both mobile and desktop screens as you can see from the video below so make sure you adjust the style of your project to remove the horizontal scroll. 

  [Screencast from 03-04-2023 01:00:20 PM.webm](https://user-images.githubusercontent.com/92303891/222904029-0fe72279-d7fa-411c-bf53-3901761dc951.webm)

[OPTIONAL] Please make sure that you set the `margin` of the page body to `0` to remove the white space around the body. :pray: You can do this :point_down: 
  ```css
  * {
    margin: 0;
  }
  ```

**Review by Owusu-Desmond — CHANGES_REQUESTED** [review]
Hi @Innocent259  :wave: 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some changes you still need to work on before you move on to the next project but you are almost there.



 ## Highlights :+1: 
:heavy_check_mark: Good commit history.
:heavy_check_mark: No linter errors.
:heavy_check_mark: Your project is well documented


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css (hunk 2 above)
Comment (Innocent259) [reply style.css:9]: Hi @jerryowusu; 
Can you please help I'm getting stuck for resolving that scroll in mobile version

**Review by emmiiorji — APPROVED** [review]
Hi @Innocent259, 

# Status: Approved ✅

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

<div align='center'>

</div>

## Good Highlights✔️

- Requested change has been implemented

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**